src/core_init_atmosphere/Registry.xml

```diff
@@ -940,6 +943,9 @@
                 <var name="xtime" type="text" dimensions="Time" units="YYYY-MM-DD_hh:mm:ss"
                      description="Model valid time"/>
 
+                <var name="Time" type="real" dimensions="Time" units="Seconds since YYYY-MM-DD hh:mm:ss"
+                     description="CF convention compliant model valid time" standard_name="time"/>
```

Comment:
Similar comment as in `core_atmosphere/Registry.xml` regarding the capitalization of `Seconds` and the description.